OpenSim/Common/DataTable.h

```diff
@@ -341,38 +485,151 @@ class DataTable_ : public AbstractDataTable {
         return _indData;
     }
 
-    /** Get dependent column.                                                 
+    /** Get dependent column at index.
 
-    \throws ColumnIndexOutOfRange If index is out of range.                   */
+    \throws ColumnIndexOutOfRange If index is out of range for number of columns
+                                  in the table.                               */
     VectorView getDependentColumnAtIndex(size_t index) const {
```

Comment:
Maybe use "dependent" for column label accessors as well, to be consistent with accessing the data.